pkg/workload/schemachange/optype_string.go

```diff
@@ -0,0 +1,49 @@
+// Code generated by "stringer -type=opType"; DO NOT EDIT.
```

Comment:
@ajwerne I had to drop enum related logic from the schemachange workload to make it 20.1 compatible. The squash hides the compatibility changes I had to make - would you prefer this to be an independent PR where I add masters version of schemachange, and then make the compatibility tweaks?